Entities/HDFirearm.h

```diff
@@ -427,6 +427,101 @@ ClassInfoGetters
 
 	void SetMuzzleOffset(Vector newOffset) override { m_MuzzleOff = newOffset; }
 
+		/// <summary>
+		/// Gets this HeldDevice's Fire Sound. Ownership is not transferred!
+		/// </summary>
+		/// <returns>The SoundContainer for this HeldDevice's Fire Sound.</returns>
+		SoundContainer * GetFireSound() const { return m_FireSound; }
+
+		/// <summary>
+		/// Sets this HeldDevice's Fire Sound.
+		/// </summary>
+		/// <param name="newSound">The new SoundContainer for this HeldDevice's Fire Sound</param>
```

Comment:
See comments in ACrab.h about `*` placement, method comment capitalization and `.`s at the end of method comments.